seqr/views/apis/data_manager_api.py

```diff
@@ -289,6 +292,8 @@ def _save_sample_data(sample_guid, sample_data):
     except ValueError as e:
         return create_json_response({'error': str(e)}, status=400)
 
+    mv_file_to_gs(f'{file_dir}/*', f'{TEMP_GS_BUCKET}/{file_name_prefix}', request.user)
```

Comment:
could you write the files directly to gcs without saving them to the temp directory first?

Reply:
Unfortunately no, that would be way to slow. This endpoint is already pushing too slow on a local filesystem, having to create and independent transaction to cloud storage for every row would be prohibitively slow